cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -256,87 +272,256 @@ The response will be the same JSON array with additional parameters for each of
 
    * - Parameter
      - Description
-   * - ``"status"``
-     - Maps to the status of an individual JSON object's queried program
-       if the query is valid and the program was found.
+   * - ``"appId"``
+     - Name of the application being called
+   * - ``"programType"``
+     - One of ``flow``, ``mapreduce``, ``spark``, ``workflow``, ``worker``, or ``service``
+   * - ``"programId"``
+     - Name of the *flow*, *MapReduce*, *Spark*, *workflow*, *worker*, or *custom service*
+       being called
    * - ``"statusCode"``
-     - The status code from retrieving the status of an individual JSON object.
+     - The status code from starting an individual JSON object.
    * - ``"error"``
-     - If an error, a description of why the status was not retrieved (the specified program was not found, etc.)
+     - If an error, a description of why the program could not be started (the specified program was not found, etc.)
 
-The ``status`` and ``error`` fields are mutually exclusive meaning if there is an error,
-then there will never be a status and vice versa.
```

Comment:
git decided to diff this section for the multi start command (which doesn't return status), with the section for the multi status command.  This comment still exists in the section for getting the status of multiple programs